apps/portals/cancercomplexity/src/pages/CCKPSearchPage.tsx

```diff
@@ -94,14 +96,28 @@ export function CCKPSearchPage(props: CCKPSearchPageProps) {
         searchPageTabs[tabIndex].count = newCount
         setSearchPageTabsState([...searchPageTabs])
       }
+      // PORTALS-3382: If no tab is selected and all counts have been set, then redirect to the item with the highest count
+      const allCountsSet = searchPageTabs.every(tab => tab.count !== undefined)
+      if (selectedTabIndex == undefined && allCountsSet) {
+        // navigate to the tab that has the highest count
+        const maxCountTab = searchPageTabs.reduce((max, tab) =>
+          max === null || tab.count! > max.count! ? tab : max,
+        )
+        navigate({
+          pathname: `/Search/${maxCountTab.path}`,
+          search: location.search,
+        })
+      }
     },
-    [searchPageTabsState],
+    [searchPageTabsState, selectedTabIndex],
   )
   // on search field value update, update the special search parameter FTS_SEARCH_TERM, which the QueryWrapperPlotNav will load as the search term
   return (
     <Box sx={{ p: { xs: '10px', lg: '50px' } }}>
-      <PortalFullTextSearchField />
-      <PortalSearchTabs tabConfig={searchPageTabsState} />
+      <PortalFullTextSearchField disabled={selectedTabIndex == undefined} />
+      {selectedTabIndex != undefined && (
+        <PortalSearchTabs tabConfig={searchPageTabsState} />
+      )}
```

Comment:
Ah, so you effectively won't even show the tabs until you have a query. This order of interactions is making more sense to me